frontend/SettingsPane.js

```diff
@@ -50,6 +50,11 @@ class SettingsPane extends React.Component {
     const doc = ReactDOM.findDOMNode(this).ownerDocument;
     // capture=true is needed to prevent chrome devtools console popping up
     doc.addEventListener('keydown', this._key, true);
+
+    const lastSearch = localStorage.getItem('searchText') || '';
+    if (lastSearch !== '') {
+      this.props.onChangeSearch(lastSearch);
```

Comment:
I don't think this is the best place to do this 😄 We should re-initialize this value in `Store` so we can avoid rendering multiple times